go/packages/packages.go

```diff
@@ -40,11 +40,44 @@ var (
 	noarch = osinfo.Architecture("noarch")
 )
 
+// Packages is a selection of packages based on their manager.
+type Packages struct {
+	Yum    []PkgInfo    `json:"yum,omitempty"`
+	Rpm    []PkgInfo    `json:"yum,omitempty"`
+	Apt    []PkgInfo    `json:"apt,omitempty"`
+	Deb    []PkgInfo    `json:"apt,omitempty"`
+	Zypper []PkgInfo    `json:"zypper,omitempty"`
+	Gem    []PkgInfo    `json:"gem,omitempty"`
+	Pip    []PkgInfo    `json:"pip,omitempty"`
+	GooGet []PkgInfo    `json:"googet,omitempty"`
+	WUA    []WUAPackage `json:"wua,omitempty"`
+	QFE    []QFEPackage `json:"qfe,omitempty"`
+}
+
 // PkgInfo describes a package.
 type PkgInfo struct {
 	Name, Arch, Version string
 }
 
+// WUAPackage describes a Windows Update Agent package.
+type WUAPackage struct {
+	Title          string
+	Description    string
+	Categories     []string
```

Comment:
As per our conversation, let's also add a non-localized category id in the form of "Security", "Critical", etc.